Plugins/JavaCompilerPlugin/Configuration.swift

```diff
@@ -0,0 +1,51 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift.org project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift.org project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+typealias JavaVersion = Int
+
+/// Configuration for the Java2Swift translation tool, provided on a per-target
+/// basis.
+///
+/// Note: there is a copy of this struct in the Java2Swift library. They
```

Comment:
Yeah, or maybe a makefile target to copy them from a SSOT.

Reply:
I’d love to not have to use make in this build but I guess for such stuff maybe we will still… just because we already have swift gradle AND make so it’d be nice to just have two builds to worry about…

but i think it’d be okey to use make just for some simple automation but let actual “building” to swiftpm and gradle. So your idea here may be good :)

Reply:
Either way, right now the copy is fine